arch/arm64/configs/aosp_nile_discovery_defconfig

```diff
@@ -733,3 +734,105 @@ CONFIG_CRYPTO_AES_ARM64_NEON_BLK=y
 CONFIG_CRYPTO_CRC32_ARM64=y
 CONFIG_QMI_ENCDEC=y
 CONFIG_WQ_POWER_EFFICIENT_DEFAULT=y
+
+# hybris
+CONFIG_AUTOFS4_FS=y
+CONFIG_BLK_CGROUP=y
+CONFIG_BT_RFCOMM=y
+CONFIG_CGROUP_DEVICE=y
+CONFIG_CGROUP_MEM_RES_CTLR=y
+CONFIG_CGROUP_MEM_RES_CTLR_KMEM=y
+CONFIG_CGROUP_MEM_RES_CTLR_SWAP=y
```

Comment:
I reverted the last commit and aligned the config with pioneer config.

Reply:
Thank you. In the commit you just reverted I've seen PROC_FS.
Just to doublecheck: have you run mer-kernel-check on the output config (.config) ? Because one shouldn't run it on a shortened defconfig like this.
I'm asking this because CONFIG_PROC_FS was already in the output config (and on device via `zcat /proc/config.bz | grep PROC_FS`) without any other related modifications

Reply:
It seems the CONFIG_MEMCG flags are already enabled in the config so you can just remove those unused CONFIG_CGROUP_MEM_RES_CTLR=y
CONFIG_CGROUP_MEM_RES_CTLR_KMEM=y
CONFIG_CGROUP_MEM_RES_CTLR_SWAP=y